symphony-bdk-http/symphony-bdk-http-webclient/src/test/java/com/symphony/bdk/http/webclient/ApiClientBuilderWebClientTest.java

```diff
@@ -37,13 +46,26 @@ void setUp() throws IOException {
 
   @Test
   void buildTest() {
+    Logger logger = (Logger) LoggerFactory.getLogger(ApiClientBuilderWebClient.class);
+    ListAppender<ILoggingEvent> listAppender = new ListAppender<>();
+    listAppender.start();
+    logger.addAppender(listAppender);
+
     builder.withTrustStore(truststore, "changeit");
     builder.withKeyStore(keystore, "password");
 
     ApiClient apiClient = builder.build();
 
     assertEquals(apiClient.getClass(), ApiClientWebClient.class);
     assertEquals(apiClient.getBasePath(), "test-base-path");
+
+    // assert logs about truststore entries
+    List<ILoggingEvent> logsList = listAppender.list;
+    assertFalse(logsList.isEmpty(), "The list of log entries should not be empty");
+    assertNotNull(logsList.get(0), "At least on entry in the list, should not be null");
+    assertEquals("Your custom truststore contains {} entries :", logsList.get(0).getMessage(),
```

Comment:
the 2 asserts before are redundant IMHO as the test would fail if there is no entry